content/e20co/drone-marathon/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-author: "dellintosh"
 date: 2017-01-06T18:00:00-06:00
-image: "e20co/drone-marathon"
-repo: "e20co/drone-marathon"
-tags: [ publish, marathon ]
+title: Marathon
+author: dellintosh
+tags: [ deploy, marathon ]
 logo: marathon.svg
-title: marathon
+repo: expansioncap/drone-marathon
```

Comment:
So there are actually a few repos, and I am not sure which one is most up-to-date:

dellintosh/drone-marathon
expansioncap/drone-marathon
drone-plugins/drone-marathon

Reply:
https://github.com/expansioncap/drone-marathon/blob/master/.drone.yml this seems to be the right one.